dbms/src/Storages/KVStore/MultiRaft/RegionData.cpp

```diff
@@ -61,6 +68,14 @@ size_t RegionData::insert(ColumnFamilyType cf, TiKVKey && key, TiKVValue && valu
     }
     case ColumnFamilyType::Default:
     {
+        // TODO(Spill) Find out whether it is a large txn, then use `insertWithTs`.
+        auto use_large = [&]() {
+            auto delta = large_default_cf.insert(std::move(key), std::move(value), mode);
+            cf_data_size += delta;
+            reportAlloc(delta);
+            return delta;
+        };
+        fiu_do_on(FailPoints::force_write_to_large_txn_default, { return use_large(); });
```

Comment:
```suggestion
            fiu_do_on(FailPoints::force_write_to_large_txn_default, {
            auto delta = large_default_cf.insert(std::move(key), std::move(value), mode);
            cf_data_size += delta;
            reportAlloc(delta);
            return delta;
        });
```

Reply:
Emmmm, `use_large` is expected to be formal logic in later PR.
The idea it that we are going to decide which cf we will write the current data into.